Formula/g/go-size-analyzer.rb

```diff
@@ -0,0 +1,51 @@
+class GoSizeAnalyzer < Formula
+  desc "Analyzing the dependencies in compiled Golang binaries"
+  homepage "https://github.com/Zxilly/go-size-analyzer"
+  url "https://github.com/Zxilly/go-size-analyzer/archive/refs/tags/v1.0.4.tar.gz"
+  sha256 "c92ebee6298fc1c6b25e5ea2ed0324abd047ef0ac3dc80b110030562e93f89ae"
+  license "AGPL-3.0-only"
+  head "https://github.com/Zxilly/go-size-analyzer.git", branch: "master"
+
+  depends_on "go" => [:build, :test]
```

Comment:
sounds like it should be runtime dep (see garble)

Reply:
go compile was not necessary at runtime. gsa analyze binary direct and got the necessary info builtin.

Reply:
There is a `bloaty` with similar functionality which not required.gcc at runtime.

Reply:
got you, thanks!